### PR TITLE
JAMES-2671 Remove duplicated Swift tests and replace general BlobStore by AWS S3

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

**Comment by blackheaven** [comment]
```
 [ERROR]   ReindexingWithEventDeadLettersTest.indexationShouldBeFailingWhenElasticSearchContainerIsPaused:128 » JsonPath
 [ERROR]   ReindexingWithEventDeadLettersTest.lambda$redeliverShouldCleanEventDeadLetter$6:149 » JsonPath
```

test this please

**Comment by blackheaven** [comment]
```
 [ERROR] yahooBase64CalendarAttachmentShouldBePublishedInMQWhenMatchingWorkflowConfiguration  Time elapsed: 385.185 s  <<< ERROR!
 org.awaitility.core.ConditionTimeoutException: Condition with lambda expression in org.apache.james.utils.IMAPMessageReader was not fulfilled within 60 seconds.
 	at org.apache.james.transport.mailets.ICSAttachmentWorkflowTest.yahooBase64CalendarAttachmentShouldBePublishedInMQWhenMatchingWorkflowConfiguration(ICSAttachmentWorkflowTest.java:712)
```

test this please

**Comment by blackheaven** [comment]
> We don't keep a test or two to check that the happy path works for swift?

It's a BlobStore, as long as you know how to deal with one, it should be ok.
IMO, we should not test jclouds.

**Comment by chibenwa** [comment]
> It's a BlobStore, as long as you know how to deal with one, it should be ok.
IMO, we should not test jclouds.

But implementation can vary in subtle ways. Having ~5 integration tests on swift anyway could help

**Comment by blackheaven** [comment]
RabbitMQSwiftFilterTest?

**Comment by trantienduchn** [comment]
I think you can keep all swift tests and remove BasicFeature tag on them, keep swift tests doesn't hurt me anyway

**Comment by blackheaven** [comment]
It means letting all Cucumber tests, that's too much IMO.

**Comment by blackheaven** [comment]
Aborted

test this please

File: server/protocols/jmap-integration-testing/rabbitmq-jmap-integration-testing/src/test/java/org/apache/james/jmap/rabbitmq/RabbitMQSwiftFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.apache.james.CassandraRabbitMQSwiftJmapTestRule;
+import org.apache.james.DockerCassandraRule;
+import org.apache.james.GuiceJamesServer;
+import org.apache.james.jmap.methods.integration.FilterTest;
+import org.apache.james.mailbox.cassandra.ids.CassandraId;
+import org.apache.james.mailbox.model.MailboxId;
+import org.junit.ClassRule;
+import org.junit.Rule;
+
+public class RabbitMQSwiftFilterTest extends FilterTest {
```
Comment (rouazana) [inline server/protocols/jmap-integration-testing/rabbitmq-jmap-integration-testing/src/test/java/org/apache/james/jmap/rabbitmq/RabbitMQSwiftFilterTest.java]: why did you choose this one?

Comment (blackheaven) [reply server/protocols/jmap-integration-testing/rabbitmq-jmap-integration-testing/src/test/java/org/apache/james/jmap/rabbitmq/RabbitMQSwiftFilterTest.java]: I've got a :+1: on https://github.com/linagora/james-project/pull/2284#issuecomment-479859314

Comment (rouazana) [reply server/protocols/jmap-integration-testing/rabbitmq-jmap-integration-testing/src/test/java/org/apache/james/jmap/rabbitmq/RabbitMQSwiftFilterTest.java]: Yes, I saw that, but why did you suggest this one at first?

Comment (blackheaven) [reply server/protocols/jmap-integration-testing/rabbitmq-jmap-integration-testing/src/test/java/org/apache/james/jmap/rabbitmq/RabbitMQSwiftFilterTest.java]: Do not look that big, do not look that small

**Pull request merged**